src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: string,
+  color: string,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0
+      || b <= 0
+      || c <= 0
+      || a >= (b + c)
+      || b >= (a + c)
+      || c >= (a + b)) {
+      throw new Error('Error: check the data');
```

Comment:
If you throw an error try to provide its detailed description in error message. and fix it in all task